src/shared/util.js

```diff
@@ -636,9 +636,13 @@ class FeatureTest {
     ) {
       return shadow(this, "platform", {
         isMac: navigator.platform.includes("Mac"),
+        isWindows: navigator.platform.includes("Win"),
+        isFirefox:
+          (typeof PDFJSDev !== "undefined" && PDFJSDev.test("MOZCENTRAL")) ||
+          navigator.userAgent.includes("Firefox"),
```

Comment:
Is this guaranteed to be safe, given everywhere that this code might run?
Please make this code a bit more defensive, to prevent any future bugs.
```suggestion
          (typeof navigator?.userAgent=== "string" && navigator.userAgent.includes("Firefox")),
```